tests/contracts/solidity/RandomnessLotteryDemo.sol

```diff
@@ -124,7 +124,7 @@ contract RandomnessLotteryDemo is RandomnessConsumer {
     {
         /// Because this contract can only perform 1 random request at a time,
         /// We only need to have 1 required deposit.
-        uint256 requiredDeposit = 1000000000000000000; // TODO replace with randomness.requiredDeposit();
+        uint256 requiredDeposit = randomness.requiredDeposit();
```

Comment:
@crystalin I replaced 10 ^ 18 with randomness.requiredDeposit()(which is set equal to `ConstU128<{ 1 * currency::UNIT * currency::SUPPLY_FACTOR }>` in moonbase)

This caused the randomness tests in this PR https://github.com/PureStake/moonbeam/pull/1748#issuecomment-1216588882 to fail once the contract was correctly regenerated. 

I am trying to fix it here, but am not sure what the correct fix was (maybe some conversion, needs to happen?) https://github.com/PureStake/moonbeam/pull/1753

Reply:
Fixed via https://github.com/PureStake/moonbeam/pull/1748/commits/964fe353fdcc29975159996d5811948c297f0fa8